DOCS/man/input.rst

```diff
@@ -3438,6 +3444,35 @@ Property list
 ``current-ao``
     Current audio output driver (name as used with ``--ao``).
 
+``clipboard`` (RW)
+    Access to the system clipboard as a key/value map of types and data.
+
+    Sub-paths can be accessed directly; e.g. ``clipboard/text`` can be read, written,
+    or observed.
+
+    The top-level object itself can be written directly as a shortcut; the intended
+    type will be guessed.
+
+    Converting this property to a string will give a JSON representation of all types.
```

Comment:
I don't think we have any other properties that do this? What's the intended use case for this behavor?

Reply:
Everything that uses `m_property_read_sub` does this (this doesn't use that to avoid reading every clipboard whenever a single one is requested, though maybe I could add a callback feature to `m_sub_property` to address that).

Reply:
The manual doesn't mention this for any other properties so that might be what tripped me up.